### PR TITLE
[SYCLomatic] Refine the description of the two build script options

### PR DESCRIPTION
**Pull request by tomflinda**
Signed-off-by: chenwei.sun <chenwei.sun@intel.com>

### CODE REVIEW

File: clang/lib/DPCT/DPCT.cpp [hunk 1]
```diff
@@ -1132,7 +1139,10 @@ int runDPCT(int argc, const char **argv) {
                      DpctGlobalInfo::getAnalysisScope(),
                      AnalysisScopeOpt.getNumOccurrences());
 
-    if (!MigrateCmakeScriptOnly &&
+    //printf(" ###################### [%d]\n", static_cast<unsigned int>(DpctGlobalInfo::getBuildScript()));
```
Comment (zhiweij1) [inline clang/lib/DPCT/DPCT.cpp]: remove

**Review by zhiweij1 — APPROVED** [review]

File: clang/lib/DPCT/DPCT.cpp (hunk 1 above)
Comment (tomflinda) [reply clang/lib/DPCT/DPCT.cpp]: done

File: clang/lib/Tooling/CommonOptionsParser.cpp [hunk 2]
```diff
@@ -182,8 +182,8 @@ OPT_TYPE OPT_VAR(OPTION_NAME, __VA_ARGS__);
 #ifdef SYCLomatic_CUSTOMIZATION
   bool IsMigrateCmakeScriptOnlySpecified = false;
   for (auto i = 0; i < argc; i++) {
-    int Res1 = strcmp(argv[i], "--migrate-cmake-script-only");
-    int Res2 = strcmp(argv[i], "-migrate-cmake-script-only");
+    int Res1 = strcmp(argv[i], "--migrate-build-script-only");
+    int Res2 = strcmp(argv[i], "-migrate-build-script-only");
```
Comment (zhimingwang36) [inline clang/lib/Tooling/CommonOptionsParser.cpp:186]: L185 and L186 are same?

Comment (tomflinda) [reply clang/lib/Tooling/CommonOptionsParser.cpp:186]: > L185 and L186 are same?
The difference is between "--" and "-"

**Comment by zhimingwang36** [comment]
LGTM

**Pull request merged**